README.md

```diff
@@ -1 +1,583 @@
-# microservices-observability
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/microservices-observability.svg?branch=master)](https://travis-ci.org/ballerina-guides/microservices-observability)
+
+# Observing Ballerina Microservices
+
+Observability of services is useful to retrieve internal information for easier troubleshooting and system status analysis. Ballerina allows to observe its services using Monitoring, Logging and Distributed tracing methods, with the use of other external tools.
```

Comment:
Minor improvement:
"Observability of services is useful to retrieve internal information for easier troubleshooting and system status analysis. Ballerina allows you to observe its services via monitoring, logging and distributed tracing methods that use external tools."